go/pkg/command/command.go

```diff
@@ -0,0 +1,273 @@
+// Package command defines common types to be used with remote execution.
```

Comment:
Can we not refer to remote execution here? This is for command execution, local or remote, correct?